### PR TITLE
feature/notifications

### PR DESCRIPTION
**Pull request by judah059**

### CODE REVIEW

File: spray/notifications/send_notifications.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+            '',
+            from_email,
+            # to:
+            to
+        )
+        msg.attach_alternative(
+            email_html_message,
+            "text/html",
+        )
+        msg.send()
```
Comment (Akhtyrtsev) [inline spray/notifications/send_notifications.py:36]: This should be just a common method not decorated as celery app task, this method will be used inside celery app task

File: spray/notifications/send_notifications.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
+from django.template.loader import render_to_string
+from django.conf import settings
+from django.core.mail import EmailMultiAlternatives
+from config.celery_app import app as celery_app
+
+
+class Notifications:
+    def __init__(self, context=None, template=None, to=None, title=None):
```
Comment (Akhtyrtsev) [inline spray/notifications/send_notifications.py:8]: can you add notification_type for future uses(email, push)

File: spray/notifications/send_notifications.py [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+        self.context = context
+        self.template = template
+        self.to = to
+        self.title = title
+
+    @celery_app.task
+    def send_mail(self):
```
Comment (Akhtyrtsev) [inline spray/notifications/send_notifications.py:15]: this should be protected method for inner usage of this class

Comment (judah059) [reply spray/notifications/send_notifications.py:15]: You said this method will be used in celery task (already used), but also you said for inner usage. May i need create new method for usage in celery task?

**Pull request closed without merging**